books/RayTracingInOneWeekend.html

```diff
@@ -3400,25 +3405,42 @@
 
 </div>
 
-<div class='together'>
-Attenuation is always 1 -- the glass surface absorbs nothing. If we try that out with these
-parameters:
+Attenuation is always 1 -- the glass surface absorbs nothing.
 
-    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ Highlight
+If we render the prior scene with the new `dielectric::scatter()` function, we see … no change. Huh?
+
+Well, it turns out that given a sphere of material with an index of refraction greater than air,
+there's no incident angle that will yield total internal reflection -- neither at the ray-sphere
+entrance point nor at the ray exit. This is due to the geometry of spheres, as a grazing incoming
+ray will always be bent to a smaller angle, and then bent back to the original angle on exit.
```

Comment:
I feel that a figure demonstrating the path of a ray and the angles of reflection/refraction (such as the one in #900) would go a long way here.

Reply:
_Sigh_. https://psgraphics.blogspot.com/2020/11/how-to-write-paper-when-you-suck-at.html

I have a huge embarrassing weakness — I am much more comfortable writing lots of text than I am generating illustrations and other graphics. Ironic, but there it is.

I think you're absolutely right. Created a new issue #1444 — feel free to jump on this if you wish.

Reply:
Sounds good, I think I can take a crack at it. Thank you for the link to the blog post, it's a great read.